XCUITests/TopTabsTest.swift

```diff
@@ -24,6 +24,7 @@ class TopTabsTest: BaseTestCase {
         waitUntilPageLoad()
         waitForTabsButton()
         // The tabs counter shows the correct number
+        waitForExistence(app.buttons["Show Tabs"])
```

Comment:
This does not seem to work on Bitrise, this test is always failing now...I have been running your branch there (see [logs](https://app.bitrise.io/build/326cc9a4bef4f79b#?tab=log)) 

What you could do is to increase the waiting time in the `waitForTabsButton()`which is doing the wait for that screen to be shown